Documentation/schemas/profile-manifest-v1.md

```diff
@@ -30,13 +30,15 @@ Profile manifest v1 includes a new field:
 - value/images/#/name: string, compatible with OCI image name specs.
   Name of the image to unpack.
 - value/images/#/reference: string, compatible with OCI image reference specs.
-  Referenced image will be locally looked up as a file named `${name}:${reference}.torcx.tgz`.
+  Referenced image will be locally looked up as a file named
+  `${name}:${reference}.torcx.${format}` where `format` may be either `tgz` or
+  `squashfs`. If both exist, either may arbitrarily be chosen.
```

Comment:
IMHO the last sentence is somehow problematic. Shouldn't we prefer "deterministically" and "squashfs"?
This would also be reflected in `ArchiveFor` logic.